tsconfig.json

```diff
@@ -15,7 +15,10 @@
     ],
     "baseUrl": "../",
     "noImplicitAny": false,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+
+    "esModuleInterop": false,
+    "allowSyntheticDefaultImports": true
```

Comment:
Document in the PR that `allowSyntheticDefaultImports` is necessary to use i18next and that this flag must be added to **all** plugins.

Reply:
@oltionchampari Please create a generator issue so that we update the two flags in the corresponding templates.